p5js/00_ImageClassification_MultipleImages/sketch.js

```diff
@@ -0,0 +1,82 @@
+/* ===
+ML5 Example
+00_ImageNet_Simple
+Simple Image Classification using p5.js
+=== */
+
+// Initialize the ImageNet method with the MobileNet model.
+const classifier = new ml5.ImageClassifier('MobileNet');
+
+let img;
+current_index = 0;
+all_images = []
+display = true;
+display_time = 750;
+predictions = []
+
+function append_images(){
+    for (var i in data.images){
+        img_path = data.images[i];
+        all_images.push(get_image_path(img_path));
+    }
+}
+
+function preload(){
+    data = loadJSON('/assets/data.json', append_images);
```

Comment:
If `loadJSON()` is used in `preload()` then there's no need for a callback. You can just call `append_images()` in `setup()`.